libs/features/commerce/test/mocks/literals.json

```diff
@@ -0,0 +1,12 @@
+{
```

Comment:
do we need mock literals?
Since we already have all the literals copied in Milo?

Reply:
@yesil

Having them mocked gives more flexibility to tests. It makes easier to find literals used in tests and customise them.